components/site/src/lib.rs

```diff
@@ -901,6 +901,24 @@ impl Site {
             }
         }
 
+        if section.meta.generate_feed {
+            let library = &self.library.read().unwrap();
+            let pages = section
+                .pages
+                .iter()
+                .map(|k| library.get_page_by_key(*k))
+                .collect();
+            self.render_feed(
+                pages,
+                Some(&PathBuf::from(&section.path[1..])),
```

Comment:
why the [1..]?

Reply:
Because section.path start with a "/" which means it will try to render the feed at the root of the filesystem instead of the root of the site directory.